osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@SoykaDaimler This information would also be available with a "position+orientation+contour" definition, you just need to calculate it. Sure, the tool could provide the oultine of an object in world coordinates or sensor coordinates but that is a redundant information because you still need "(reference point)position+orientation".

I also would suggest to discuss the reference point of vehicles again. In my opinion it makes no sense to have different reference point definitions, depending on whether a vehicle is a "Ego" or not. OpenSCENARIO until now does not provide any means to distinguish between vehicles that are an "Ego" and other vehicles. Reference point of a vehicle is always center of rear axis on ground in neutral load conditions.
@raueATconti the whole coordinate topic might be worth discussing it on the workshop.